packages/liveblocks-react-comments/README.md

```diff
@@ -27,9 +27,10 @@ and scale collaborative applications with zero configuration, no maintenance
 required.
 
 > [!IMPORTANT]  
-> Comments is currently in private beta and is only available to a select group
-> of users. If you want to participate in the beta,
-> [sign up for the waitlist](https://forms.gle/xgp54aTMsoH8hrjU9).
+> Comments is currently in beta.If you have any questions or feedback, please
+> reach out to us via the dedicated Discord channel,
+> [#comments-beta](https://discord.com/channels/913109211746009108/1126614905160749097).
+> We’d love to hear from you.
```

Comment:
I think we can remove this one entirely.

Reply:
Sorry for replying late. I was focused on other things last week. Were you suggesting to remove the whole text from the README or just the last line `We'd love to hear you` from the text?

Reply:
The whole text, I think we say it enough times on the site and the docs.

Reply:
Thanks! and done